src/frontend/js/components/SearchSuggestField/SearchSuggestField.spec.tsx

```diff
@@ -1,14 +1,14 @@
-import '../../testSetup';
+import 'testSetup';
 
 import { fireEvent, render, wait } from '@testing-library/react';
 import fetchMock from 'fetch-mock';
 import React from 'react';
 import { IntlProvider } from 'react-intl';
 
-import { CourseSearchParamsContext } from '../../data/useCourseSearchParams/useCourseSearchParams';
+import { CourseSearchParamsContext } from 'data/useCourseSearchParams/useCourseSearchParams';
 import { SearchSuggestField } from './SearchSuggestField';
```

Comment:
In this commit some paths are still relative. I know they only concern tests and the tested component for for a seek of consistency we should maybe also use absolute paths ?

Reply:
I think generally absolute paths are better but when we're importing stuff from the same directory IMO it's part of the same module and it makes more sense to import it directly. The same goes for tests.

Now of course this is debatable but I think simplicity trumps consistency here.